src/test/java/task3/PersonalDataTest.java

```diff
@@ -0,0 +1,33 @@
+package task3;
+
+import lambdastreams.task3.Person;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+class PersonalDataTest {
+
+    @Test
+    void adultPersons() {
+        List<String> right = Arrays.asList("PETER", "PAUL", "ARNOLD", "AARON");
```

Comment:
Please rename this variable to smth like "expectedList"

Reply:
Done